src/SecretLobby.Core/Entities/IServerImpl.cs

```diff
@@ -0,0 +1,148 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace SecretLobby
+{
+    [JsonObject(
+        MemberSerialization = MemberSerialization.OptIn,
+        MissingMemberHandling = MissingMemberHandling.Ignore,
+        ItemNullValueHandling = NullValueHandling.Include,
+        ItemRequired = Required.Default,
+
+        IsReference = false,
+        ItemIsReference = false,
+        Id = "SecretLobby.IServer"
+        )]
+    public sealed class IServerImpl : IServer
```

Comment:
A non-interface shouldn't be prefixed with "I"

Reply:
Yes, my mistake, it will be fixed